app/src/main/java/ch/epfl/cs311/wanderwave/model/utils/BeaconPlacer.kt

```diff
@@ -71,12 +120,18 @@ fun computeDistanceBetweenBeacons(newBeacons: MutableList<Beacon>, beacons: List
  * @since 2.0
  * @last update 2.0
  */
-fun findRandomBeacon(location: Location, newBeacons: MutableList<Beacon>, it: Int) {
+fun findRandomBeacon(
+    location: Location,
+    newBeacons: MutableList<Beacon>,
+    it: Int,
+    beaconRepository: BeaconRepository
+) {
   val randomLocation = randomLatLongFromPosition(location, BEACON_RADIUS)
   val newBeacon =
       Beacon(
-          id = "beacon${newBeacons.size + it}", // TODO: modify the ID!!!!!!!!
+          id = "", // TODO a modifier
```

Comment:
Just want to make sure you haven't forgotten this TODO :)

Reply:
I forgot it thx